### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by alsedi**
Recommended by GitHub, the Code of Conduct has been used.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+opensource@umain.com.
```
Comment (TheRogue76) [inline CODE_OF_CONDUCT.md:63]: Is this email setup already?

Comment (alsedi) [reply CODE_OF_CONDUCT.md:63]: yes

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+* Demonstrating empathy and kindness toward other people
```
Comment (casvanluijtelaar) [inline CODE_OF_CONDUCT.md:20]: "towards"

**Review by casvanluijtelaar — COMMENTED** [review]
very nice!

**Review by jacksonmafra-umain — APPROVED** [review]
Great work

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (alsedi) [reply CODE_OF_CONDUCT.md]: updated

**Review by TheRogue76 — APPROVED** [review]

**Pull request merged**